prison-core/src/main/java/tech/mcprison/prison/Prison.java

```diff
@@ -50,6 +50,8 @@
  */
 public class Prison implements PluginEntity {
 
+	public static final int SPIGOTMC_ORG_PROJECT_ID = 72740;
```

Comment:
Pleaase change this to the official resource.

Reply:
already did.  That's in the commits that still need to be pulled.